### PR TITLE
Feat/4401 allow to open file in a comment as content

### PR DESCRIPTION
**Pull request by asf-algoo**
#4401 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/Timeline/CommentFilePreview.jsx [hunk 1]
```diff
@@ -77,6 +77,19 @@ export const CommentFilePreview = props => {
               buttonIcon='fas fa-ellipsis-v'
               buttonTooltip={props.t('Actions')}
             >
+              {/*
+                FIXME - G.B. - 2021-04-02 - Ideally it would be good to use Link here, but since this
```
Comment (raphj) [inline frontend_lib/src/component/Timeline/CommentFilePreview.jsx:81]: maybe add a link to an issue there

**Review by raphj — APPROVED** [review]

**Review by grignards — APPROVED** [review]

**Review by Skylsmoi — APPROVED** [review]

File: frontend_lib/src/component/Timeline/CommentFilePreview.jsx (hunk 1 above)
Comment (Skylsmoi) [reply frontend_lib/src/component/Timeline/CommentFilePreview.jsx:81]: yes, FIXME and TODO comments require a link to a github issue

Comment (gbonaspetti) [reply frontend_lib/src/component/Timeline/CommentFilePreview.jsx]: fixed

**Review by raphj — APPROVED** [review]

**Review by Skylsmoi — APPROVED** [review]

**Comment by grignards** [comment]
Tested on branch.

**Pull request merged**